packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -416,7 +416,6 @@ const SwipeableDrawer = React.forwardRef(function SwipeableDrawer(props, ref) {
         }}
         anchor={anchor}
         transitionDuration={transitionDuration}
-        hideBackdrop={hideBackdrop}
```

Comment:
Hmm, it looks like this is passed through in the old version but when variant="permanent" it gets passed to a div.